.travis.yml

```diff
@@ -4,5 +4,6 @@ go:
     - 1.4
 
 before_script:
+    - go get -d -v ./...
```

Comment:
I apologize, I'm still learning the whole Go toolchain and was confused by the Travis CI build failure (https://travis-ci.org/tonistiigi/dnsdock/builds/69731233). It looked like Travis didn't have all the deps and I found this line in an early commit, so I gave it a shot.

Later, I happened to notice that there were new files under Godeps/_workspace/src/github.com/docker; I think perhaps I should have instead added those to the 'Update samalba/dockerclient dep' commit.